### PR TITLE
v0.6/v0.7/v1.0 Support

### PR DESCRIPTION
**Pull request by ccoffrin**
Currently this PR is for a preliminary code review, while we wait for dependent packages to be tagged.

@chriscoey, has requested to move from `time_ns()` to `time()`

### CODE REVIEW

**Comment by chriscoey** [comment]
failing on 1.0

File: src/Pajarito.jl [hunk 1]
```diff
@@ -18,6 +18,34 @@ module Pajarito
...
+import Compat: hasmethod
+import Compat: rmul!
+import Compat: norm
+
+# Needed for only for Julia v0.6 compatability
+if VERSION < v"0.7.0-"
+    eigen! = eigfact!
+end
+
+
```
Comment (chriscoey) [inline src/Pajarito.jl]: extra whitespace

File: src/conic_algorithm.jl [hunk 2]
```diff
@@ -477,9 +483,9 @@ function MathProgBase.optimize!(m::PajaritoConicModel)
     if m.log_level > 1
         @printf "\n%-33s" "Transforming data..."
     end
-    tic()
+    start_time_trans = time_ns()
     (c_new, A_new, b_new, cone_con_new, cone_var_new, keep_cols, var_types_new, cols_cont, cols_int) = transform_data(copy(m.c_orig), copy(m.A_orig), copy(m.b_orig), deepcopy(m.cone_con_orig), deepcopy(m.cone_var_orig), copy(m.var_types), m.solve_relax)
-    m.logs[:data_trans] += toq()
+    m.logs[:data_trans] += (time_ns() - start_time_trans)*1e-9
```
Comment (chriscoey) [inline src/conic_algorithm.jl]: don't need to multiply by 1e-9 when use time()

File: src/conic_algorithm.jl [hunk 3]
```diff
@@ -2051,7 +2060,7 @@ function add_sep_cut_exp!(m, add_cuts::Bool, r, s, t)
 
             # K* separation cut on (r,s,t) is (t/r, -2*log(exp(1)*t/2r), -2)
             u_val = t_val/r_val
-            v_val = -2.*(1. + log(u_val/2.))
+            v_val = -2 .* (1. + log(u_val/2.))
```
Comment (chriscoey) [inline src/conic_algorithm.jl]: here and below the "." was supposed to make 2 a float, not to broadcast the *
```suggestion
            v_val = -2.0 * (1.0 + log(u_val/2.0))
```

File: src/conic_algorithm.jl [hunk 4]
```diff
@@ -251,8 +251,11 @@ function MathProgBase.loadproblem!(m::PajaritoConicModel, c, A, b, cone_con, con
             error("Some indices in a constraint cone do not correspond to indices of vector b\n")
         end
 
-        inds_con[inds] += 1
+        for i in inds
```
Comment (mlubin) [inline src/conic_algorithm.jl]: `inds_con[inds] .+= 1` should work

File: src/conic_algorithm.jl [hunk 5]
```diff
@@ -267,7 +270,9 @@ function MathProgBase.loadproblem!(m::PajaritoConicModel, c, A, b, cone_con, con
             error("Some indices in a variable cone do not correspond to indices of vector c\n")
         end
 
-        inds_var[inds] += 1
+        for i in inds
+            inds_var[i] += 1
```
Comment (mlubin) [inline src/conic_algorithm.jl]: `inds_var[inds] .+= 1`

File: src/conic_algorithm.jl (hunk 4 above)
Comment (ccoffrin) [reply src/conic_algorithm.jl]: For some reason this variant resulted in `ErrorException("Some indices in vector c do not correspond to indices of a variable cone\n")` in Julia v0.7 and v1.0.

**Pull request merged**